src/mlpack/tests/CMakeLists.txt

```
@@ -174,6 +172,8 @@ add_executable(mlpack_catch_test
   serialization_catch.hpp
   test_catch_tools.hpp
   image_load_test.cpp
+  linear_regression_test.cpp
+  main_tests/linear_regression_test.cpp
   main_tests/image_converter_test.cpp
```

```suggestion
  main_tests/image_converter_test.cpp
  main_tests/linear_regression_test.cpp
```
Just to fix the alphabetical ordering. :)

---

Done